src/util.js

```diff
@@ -14,15 +15,3 @@ export function assign(obj, props) {
 	for (let i in props) obj[i] = props[i];
 	return /** @type {O & P} */ (obj);
 }
-
-/**
- * Remove a child node from its parent if attached. This is a workaround for
- * IE11 which doesn't support `Element.prototype.remove()`. Using this function
- * is smaller than including a dedicated polyfill.
- * @param {import('./index').ContainerNode} node The node to remove
- */
-export function removeNode(node) {
```

Comment:
@developit the perf regression still stands after undoing the `assign` change so I am thinking this might be related to `removeNode`